cloudnet-modules/cloudnet-modules-permission/src/main/java/de/dytanic/cloudnetcore/permissions/config/ConfigPermissions.java

```diff
@@ -41,10 +41,12 @@ public ConfigPermissions() throws Exception
             {
                 PermissionGroup member = new PermissionGroup(
                         "default",
+                        "&8",
                         "§eMember §7▎ ",
                         "§f",
                         "§e",
-                        9999,
+                        9999
```

Comment:
```suggestion
                        9999,
```

Please format your code properly